### PR TITLE
Add wrapped BNB, btcb, link, USDt for BNC

### PR DESCRIPTION
**Pull request by EnoRage**
https://bscscan.com/tokens

### CODE REVIEW

File: services/markets/coinmarketcap/mapping.go [hunk 1]
```diff
@@ -1164,6 +1164,36 @@ const mapping = `[
...
+        "id": 1975
+    },
+	{
+        "coin": 10000714,
+        "type": "token",
+ 		"token_id": "0xe9e7cea3dedca5984780bafc599bd69add087d56",
+        "id": 4687
+    },
+	{
+        "coin": 10000714,a
```
Comment (vikmeup) [inline services/markets/coinmarketcap/mapping.go:1192]: Extra a?

File: services/markets/coinmarketcap/mapping.go [hunk 2]
```diff
@@ -1164,6 +1164,36 @@ const mapping = `[
         "type": "coin",
         "id": 1839
     },
+    {
+        "coin": 10000714,
+        "type": "token",
+ 		"token_id": "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c",
+        "id": 1839
+    },
+ 	{
```
Comment (vikmeup) [inline services/markets/coinmarketcap/mapping.go:1173]: Do we have json validation file test?

**Review by vikmeup — APPROVED** [review]

File: services/markets/coinmarketcap/mapping.go (hunk 1 above)
Comment (EnoRage) [reply services/markets/coinmarketcap/mapping.go:1192]: Will apply cleanup

File: services/markets/coinmarketcap/mapping.go (hunk 2 above)
Comment (EnoRage) [reply services/markets/coinmarketcap/mapping.go:1173]: Yes, will do

File: services/markets/coingecko/base_test.go [hunk 3]
```diff
@@ -116,5 +116,5 @@ var (
...
....629Z","volume":17356592769,"market_cap":22851909019},{"coin":714,"coin_name":"BNB","type":"coin","price":{"change_24h":-0.9431,"currency":"USD","provider":"coingecko","value":16.76},"last_update":"2020-05-07T17:10:27.413Z","volume":386293713,"market_cap":2474368090},{"coin":195,"coin_name":"TRX","type":"coin","price":{"change_24h":-0.51593,"currency":"USD","provider":"coingecko","value":0.01594768},"last_update":"2020-05-07T17:10:27.759Z","volume":1697721558,"market_cap":1057389936},{"coin":60,"coin_name":"ETH","token_id":"0xb1bafca3737268a96673a250173b6ed8f1b5b65f","type":"token","price":{"change_24h":0,"currency":"USD","provider":"coingecko","value":0.00083971},"last_update":"2019-12-26T04:00:21.046Z","volume":69.52,"market_cap":0},{"coin":714,"coin_name":"BNB","token_id":"xrp-bf2","type":"token","price":{"change_24h":-0.92136,"currency":"USD","provider":"coingecko","value":0.21726},"last_update":"2020-05-07T17:14:13.364Z","volume":301.34,"market_cap":0},{"coin":195,"coin_name":"TRX","token_id":"1000451","type":"token","price":{"change_24h":2.39645,"currency":"USD","provider":"coingecko","value":0.00000808},"last_update":"2020-05-07T17:10:13.895Z","volume":7.87,"market_cap":0}]`
```
Comment (vikmeup) [inline services/markets/coingecko/base_test.go:119]: `"last_update":"2020-05-07T17:10:27.759Z"` i don't think this is needed

File: services/markets/coinmarketcap/mapping.go [hunk 4]
```diff
@@ -1164,6 +1164,36 @@ const mapping = `[
...
+        "coin": 10000714,
+        "type": "token",
+ 		"token_id": "0x55d398326f99059ff775485246999027b3197955",
```
Comment (vikmeup) [inline services/markets/coinmarketcap/mapping.go:1176]: did you consider formatting your JSON file?

**Pull request merged**